content/2023-berlin/speakers/david-calvert.md

```diff
@@ -2,8 +2,6 @@
 title: "David Calvert"
 ---
 
-## David Calvert
-
-Working at Oqton
```

Comment:
We always have that as a first line of the biography. Let's be consistent even though it's mentioned in your text too.

Reply:
Sorry, I didn't notice the pattern, just swapped :+1: